discussment-ui-wicket/src/main/java/org/danekja/discussment/ui/wicket/list/topic/TopicListPanel.java

```diff
@@ -59,11 +59,11 @@ public TopicListPanel(String id,
     protected void onInitialize() {
         super.onInitialize();
 
-        /*if (topicListModel.getObject().size() == 0) {
+        if (topicListModel.getObject().size() == 0) {
```

Comment:
This belongs to onConfigure method, not onInitialize - onConfigure checks everytime the component is updated/rendered (e.g via AJAX), onInitialize is run only once per full page load.